app/helpers/application_helper/button/schedule_run_now.rb

```diff
@@ -0,0 +1,7 @@
+class ApplicationHelper::Button::ScheduleRunNow < ApplicationHelper::Button::ButtonWithoutRbacCheck
```

Comment:
@vecerek Hm, i know I've suggested to use `ButtonWithoutRbacCheck` class, but now, when I'm trying the button, I'm getting back error message:

![screenshot from 2017-02-23 15-18-33](https://cloud.githubusercontent.com/assets/1187051/23262531/5b4175e4-f9db-11e6-956d-90b498dc010a.png)

So, the button probably can be `Basic`, as it was apparently disabled for a reason, and the only strange thing to me, is that the OOTB `admin` user does not have rights to use this feature.

Seems like other issue to me.

cc/ @dclarizio